examples/next/pages/_app.page.tsx

```diff
@@ -4,13 +4,51 @@
 import App from 'next/app';
 import { Amplify, Hub } from 'aws-amplify';
 import { Authenticator, AmplifyProvider } from '@aws-amplify/ui-react';
+import { MapProvider, useMap } from 'react-map-gl';
 
 if (typeof window !== 'undefined') {
   window['Amplify'] = Amplify;
   window['Hub'] = Hub;
 }
 
+const SignalIdleMap = () => {
+  const { default: map } = useMap();
+
+  map?.once('load', () => {
```

Comment:
Oh have not seen this before but im guessing this helps resolve multiple map renders?

Reply:
Eh, in the case of loading it could be either `map.on` or `map.once` since the load callback is only called once. I just felt that this made it clearer since we only want to set the map object on the window once.